sashimi-webapp/src/logic/processor/processorPump.js

```diff
@@ -0,0 +1,41 @@
+// Importing modules
+const cm = require("../contentManipulator/contentManipulator");
```

Comment:
All string should be wrapped with single quote `'`.

However, I had accidentally turn off that eslint rule in this branch.  
Next time we will have the eslint to replace all `"` with `'` for us.